ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/AttestationUtil.java

```diff
@@ -144,11 +182,8 @@ public static UnsignedLong get_current_epoch_boundary_attesting_balance(BeaconSt
     List<Integer> current_epoch_boundary_attester_indices =
         AttestationUtil.get_attester_indices(state, current_epoch_boundary_attestations);
 
-    // Get current_epoch_boundary_attesting_balance
-    UnsignedLong current_epoch_boundary_attesting_balance =
-        AttestationUtil.get_total_attesting_balance(state, current_epoch_boundary_attester_indices);
-
-    return current_epoch_boundary_attesting_balance;
+    return AttestationUtil.get_total_attesting_balance(
```

Comment:
You don't need to prefix the call to `get_total_attesting_balance ` with `AttestationUtil` since this code is in `AttestationUtil`.